android-runtime/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':app', ':runtime'
+rootProject.name='Sdkgen Kotlin Android Runtime'
```

Comment:
This line can be removed

Additionally, as app should be removed, it needs to be removed here